src/main/java/frc/robot/constants/IntakeConstants.java

```diff
@@ -4,4 +4,11 @@ public class IntakeConstants {
   public static final int intakeMotorID = 15;
 
   public static final double intakeMotorCurrentLimit = 5;
+
+  public static final double intakeVoltage = 3;
+  public static final double ejectVoltage = -4;
+
+  // Used in auto. Teleop is just whileTrue.
+  public static final double intakeTime = 2.5;
```

Comment:
I would name this variable `intakeTimeout` to be clear it can be terminated early. Same for ejectTime